devicon.json

```diff
@@ -2934,6 +2934,28 @@
             }
         ]
     },
+    {
+        "name": "githubcodespaces",
+        "altnames": ["GitHub Codespaces"],
+        "tags": [
+            "development",
+            "remote-development",
+            "editor",
+            "browser",
+            "cloud"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#24292e",
+        "aliases": []
+    },
     {
         "name": "gitlab",
         "altnames": [],
```

Comment:
Hi @lunatic-fox! :) I hope you’re doing well! :) I’m thinking it would be beneficial to change the label from plural to singular (“altnames” —> “altname”). Easier to read. As far the alt name itself, call it “gitlab”. It’ll be familiar to coders with disabilities who use this tool. If you have any questions or feedback on my review (this my first one for this repo. Woo! 🎉🎊), don’t hesitate to tag me in this comment section.

Reply:
Hello there @CBID2! I hope you're doing well too!

Change `altnames` to `altname` would break the structure and backwards compatibility of `devicon.json`, because it's a property key.

`altnames` is a array of strings property key, in other words, it works like a title of a list, for instance: "items", "groceries list", "tasks". They're usually in plural, even when they're empty.

`gitlab` doesn't have any alternative name: `"altnames": []`

I recently made a feature request to make Intellisense work on `devicon.json`, as many users use VS Code or GitHub Codespaces to edit the code, that way you can create a new entry object easily and have some description of each property when you hover over them.
* #1481 

I hope I got everything right!
Wish for the best!

Reply:
I like that idea @lunatic-fox! :) I use VS Code , so this would be helpful.